tests/utils/test_utils_generate.py

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: 2024 SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import os
+import subprocess
+
+
+def test_utils_gen_keypair(astarte_env_vars):
+    realm = astarte_env_vars["realm"]
+
+    arg_list = [
+        "astartectl",
+        "utils",
+        "gen-keypair",
+        realm,
+    ]
+
+    sample_data_result = subprocess.run(arg_list, capture_output=True, text=True)
+    except_output = "Keypair generated successfully"
+    assert except_output in sample_data_result.stdout
+
+
+def test_utils_gen_jwt_all_realms(astarte_env_vars):
+
+    realm = astarte_env_vars["realm"]
+    private_key_dir = _get_private_key(realm)
+
+    arg_list = ["astartectl", "utils", "gen-jwt", "all-realm-apis", "-k", private_key_dir]
+    sample_data_result = subprocess.run(arg_list, capture_output=True, text=True)
+    assert len(sample_data_result.stdout) > 0
```

Comment:
The output JWT should contain only specific claims (see [here](https://docs.astarte-platform.org/astarte/latest/070-auth.html#token-claims-and-formats)). You can use [pyjwt](https://pyjwt.readthedocs.io/en/stable/) or [python-jose](https://github.com/mpdavis/python-jose) (or any other library) to check that the correct claims were generated